ocs_sample_library_preview/SDS/SdsStream.py

```diff
@@ -234,12 +234,12 @@ def fromJson(content: dict[str, str]):
         if 'InterpolationMode' in content:
             interpolation_mode = content['InterpolationMode']
             if interpolation_mode is not None:
-                result.InterpolationMode = SdsInterpolationMode[interpolation_mode]
+                result.InterpolationMode = SdsInterpolationMode(interpolation_mode)
```

Comment:
If I'm not mistaken, this also needs to be fixed in the `SdsStreamPropertyOverride` type. Otherwise this looks good.